tv_apps/browser/js/toolbar.js

```diff
@@ -711,6 +714,26 @@ var Toolbar = {
     return this.toolbarPanel.dataset.mode;
   },
 
+  /**
+   * Open Pocket List
+   */
+  clickPocketListBlock: function toolbar_clickPocketListBlock() {
+    var pocketHomePageUri = 'https://getpocket.com/';
+    if(Browser.currentInfo.url !== null &&
+      Browser.currentInfo.url !== ''){
```

Comment:
I would change the if condition to 
```if (Browser.currentInfo.url && UrlHelper.isURL(Browser.currentInfo.url))```. Because if the parameter is null, the UrlHelper.isURL would be failed.